kadok.go

```diff
@@ -1,5 +1,24 @@
+// Kadok is a bot developed for the Discord Guild "Les petits pedestres". It aims to provide fun and useful functionnalities for the Guilde Members.
```

Comment:
is all of that relevant here since it is present in `README` ?

Reply:
This message is relevant as it is displayed as a short package description for Kadok in godoc